app/Resources/views/user/show.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>User</h1>
+
+    <table>
+        <tbody>
+            <tr>
+                <th>Id</th>
+                <td>{{ user.id }}</td>
+            </tr>
+        </tbody>
+    </table>
+
+    <ul>
+        <li>
+            <a href="{{ path('user_index') }}">Back to the list</a>
```

Comment:
zróbmy tutaj z użyciem translatora, szkoda zeby to było na żywca w twigu trzymane